### DISCUSSION TITLE
`$ sudo apt install robotpkg-example-robot-data` does not work on Ubuntu20.04 machine.

### DISCUSSION THREAD

**Discussion by syw-robotics** [stack-of-tasks/pinocchio#2140] [Ideas]
It seems that **robotpkg-example-robot-data** is not in APT repo yet. 

`$ sudo apt install robotpkg-example-robot-data` does not work for me on Ubuntu20.04, which made me spent much time to fix it but didn't work finally. 

Does any handsome guy have got an idea?

**Comment by nim65s** [comment]
Hi @SeaHI-Robot,

This is working for me. 

I understand that one can be frustrated when something is not working, but the fact that you "spent much time to fix it but didn't work finally" is not a useful piece of information for us if we want to understand you case and help you. Instead, you should provide a report detailing what you did, what result you expected, and what you get instead. Copy pasting error messages would be great too, if you had any.

Anyways, did you follow robotpkg debian packages setup process, in either https://stack-of-tasks.github.io/pinocchio/download.html or http://robotpkg.openrobots.org/debian.html ?

**Comment by syw-robotics** [comment]
Hi, Mr. Saurel!

So glad to find your reply! I apologize for my poor problem description, and thank you for your detailed reply, which taught me how to better participate in effective discussions.

I just succesfully installed `robotpkg-example-robot-data`, which turned out to be a VPN issue (I guess). 

But still, I am stucked in installing pinocchio, following https://stack-of-tasks.github.io/pinocchio/download.html or http://robotpkg.openrobots.org/debian.html. Luckily, I fixed this problem. Here comes the details, so that other people can see as a reference:

- Make `robotpkg`  added via APT;

- When runing ` sudo apt install -qqy robotpkg-py3*-pinocchio`, it shows:

![image-20240129013008930](https://github.com/stack-of-tasks/pinocchio/assets/84261527/58cb37e1-3356-4359-969d-797598e49bac)
![image-20240129013034257](https://github.com/stack-of-tasks/pinocchio/assets/84261527/e62471bf-0a99-41a7-9283-fcfabf6d1836)

- Then I tried to run `sudo apt install robotpkg-hpp-fcl`, it showed that  `assimp-util` was not installed as a dependency. After that I tried `sudo apt install assimp-utils` but failed.

-  Finally, I found this link :https://launchpad.net/ubuntu/focal/amd64/assimp-utils/5.0.1~ds0-1build1. Manually，downloaded the `.deb` file of assimp-utils which is compatible with ubuntu20.04， and run `sudo dpkg -t assimp-utils******` to install it manually. Then tried ` sudo apt install -qqy robotpkg-py3*-pinocchio`, it worked just fine finally.
